includes/class-wc-discounts.php

```diff
@@ -159,7 +159,7 @@ public function get_items_to_validate() {
 	 * @since  3.2.0
 	 * @param  string $key name of discount row to return.
 	 * @param  bool   $in_cents Should the totals be returned in cents, or without precision.
-	 * @return array
+	 * @return int
```

Comment:
In fact it returns `float` and not `int`.

Reply:
@claudiosanches thank you for pointing that. I change to  float.